src/Views/Form/Templates/Sequoia/assets/css/form.scss

```diff
@@ -1145,12 +1145,12 @@ fieldset {
 	display: none !important;
 }
 
-input[type='checkbox'],
-input[type='radio']{
-	opacity: 0 !important;
-	position: absolute !important;
-	left: -60px;
-}
+//input[type='checkbox'],
+//input[type='radio']{
+//	opacity: 0 !important;
+//	position: absolute !important;
+//	left: -60px;
+//}
```

Comment:
Let’s just remove this. It’s version controlled so we can always get it back.

Reply:
fixed: https://github.com/impress-org/givewp/compare/f9b03c6b789b...af66065616b3